### PR TITLE
New layer selector with chip style

### PR DESCRIPTION
**Pull request by fsdavid**

### CODE REVIEW

File: src/atlasComponents/sapiViews/core/atlas/smartChip/atlas.smartChip.components.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import {Component, EventEmitter, Input, Output} from "@angular/core";
```
Comment (xgui3783) [inline src/atlasComponents/sapiViews/core/atlas/smartChip/atlas.smartChip.components.ts:1]: ditto see comment from previous PR

**Review by xgui3783 — COMMENTED** [review]
You don't have to start a new branch/PR. 

if it is about the same feature, it is usually less confusing to commit on top of previous branch, so that previous comments are not lost.

File: src/atlasComponents/sapiViews/core/atlas/smartChip/atlas.smartChip.components.ts (hunk 1 above)
Comment (fsdavid) [reply src/atlasComponents/sapiViews/core/atlas/smartChip/atlas.smartChip.components.ts:1]: Currently, all 3 components for atlas, space, and parcellation are different enough to keep them separate. especially selector for parcellation

**Comment by fsdavid** [comment]
> You don't have to start a new branch/PR.
> 
> if it is about the same feature, it is usually less confusing to commit on top of previous branch, so that previous comments are not lost.

Sure, sorry for that. I do not know why, but the previous version was not building correctly. After rendering new commit, viewer was not updating, (cache clean was not working as well).

**Comment by xgui3783** [comment]
@fsdavid can you take a look at the PR #1269 

specifically, a generic smart chip: https://siibra-explorer.apps-dev.hbp.eu/feataltatpselector/storybook-static/?path=/story/components-smartchip--default

how it looks as a component (color picker should work here): https://siibra-explorer.apps-dev.hbp.eu/feataltatpselector/storybook-static/?path=/story/atlascomponents-sapiviews-core-rich-atpselector-puredumb-pureatpselector--default

and how it interacts (e.g. check if template/parc is compatible etc) https://siibra-explorer.apps-dev.hbp.eu/feataltatpselector/storybook-static/?path=/story/atlascomponents-sapiviews-core-rich-atpselector-wrapper--default

I am not 100% happy with how the UI looks either, but it should be a good way to test out UIs before pushing them to dev/prod

Can you take a look at the PR, and ideally make your implementation as a storybook story, so that we can evaluate it in an upcoming dev  meeting?

**Comment by xgui3783** [comment]
fixes #1230 #1190

**Pull request merged**